main.js

```diff
@@ -0,0 +1,29 @@
+const core = require('@actions/core');
+const { shell } = require('./shell.js');
+
+async function install() {
+    await shell(`
+    curl --proto '=https' --tlsv1.2 -sSf https://edgeguard-app.s3.us-west-1.amazonaws.com/linux/install.sh | sudo bash
```

Comment:
Not sure that this is the best way to translate shell usage to js, but, whatever works.